src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        Color someColor = Color.values()[random.nextInt(Color.values().length)];
+        return someColor.toString();
```

Comment:
use enum's `name()` instead of `toString()` try to think why it would be better 😉

Reply:
toString() can be overridden name() is final and it is ensuring that it will always return the exact name of the enum constant as declared in the enum type. Better to read code. The explanations I found.